development/build/scripts.js

```diff
@@ -834,6 +893,7 @@ function getEnvironmentVariables({ buildType, devMode, testing, version }) {
     ONBOARDING_V2: metamaskrc.ONBOARDING_V2 === '1',
     COLLECTIBLES_V1: metamaskrc.COLLECTIBLES_V1 === '1',
     TOKEN_DETECTION_V2: metamaskrc.TOKEN_DETECTION_V2 === '1',
+    ENABLE_MV3: metamaskrc.ENABLE_MV3 === '1',
```

Comment:
we should not required setting this value in the rc, it should all be set via the yarn script